pkg/types/installconfig.go

```diff
@@ -119,6 +119,12 @@ type InstallConfig struct {
 	// +kubebuilder:default=false
 	// +optional
 	FIPS bool `json:"fips,omitempty"`
+
+	// CloudCredentialsMode instructs the installer to not attempt to query the cloud permissions before attempting
+	// installation. It also passes down the desired credentials mode to the cloud-credential-operator so that it too
+	// does not attempt to query permissions.
+	// +optional
```

Comment:
this is missing the behavior of the installer when this is left empty / un-set as part of documentation.